sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/AccountSASSignatureValues.java

```diff
@@ -10,7 +10,7 @@
 /**
```

Comment:
Do you want to rename this class as well?

Reply:
Yes, I will have a separate PR for renaming all acronym related classes to minimize merge conflicts.